utils/handleToasts.ts

```diff
@@ -0,0 +1,52 @@
+import { toast } from 'react-toastify';
+
+function generateToastID() {
+  return checkIsIdOccupied(randomId());
+}
+
+function randomId() {
+  return Math.floor(Math.random() * 100000000).toString();
+}
```

Comment:
Tämä voinee olla turha funktio. `Math.floor`in pitäisi pystyä laittamaan suoraan `checkIsIdOccupied`iin argumenttina

tähän tapaan:
```ts
checkIsIdOccupied(Math.floor(Math.random() * 100000000).toString())
```

Reply:
Tein tällä tavalla ID:n määrityksen, koska tämä oli simppelein tapa :smile:

Reply:
Moderneissa selaimissa `window.crypto.randomUUID()` on myös näppärä tapa generoida uniikki id, jolloin ei tarvitse tarkistaa duplikaatteja.

Reply:
Joo, tämä oli hyvä! Sai otettua turhat funktionit pois samalla. Kiitos!! :+1: